front/src/modules/pathfinding/components/Itinerary/DisplayItinerary/Vias.tsx

```diff
@@ -12,13 +12,21 @@ import { useAppDispatch } from 'store';
 import { formatUicToCi } from 'utils/strings';
 
 import ViaStopDurationSelector from './ViaStopDurationSelector';
+import type { PathStep } from 'reducers/osrdconf/types';
 
 type ViasProps = {
   shouldManageStopDuration?: boolean;
   zoomToFeaturePoint: (lngLat?: Position, id?: string) => void;
+  pathSteps: PathStep[];
+  isPathStepInvalid: (step: PathStep | null) => boolean;
 };
 
-const Vias = ({ zoomToFeaturePoint, shouldManageStopDuration }: ViasProps) => {
+const Vias = ({
+  zoomToFeaturePoint,
+  shouldManageStopDuration,
+  pathSteps,
```

Comment:
No need to give pathSteps in this component, you can directly use the vias (from the selector getVias)

Reply:
done